File: Data Structures & Algorithms/remove-element/submission-0.java
class Solution {
    public int removeElement(int[] nums, int val) {
        int base = 0;
        int out = 0;

        while (out < nums.length) {
            if (nums[out] != val) {
                nums[base] = nums[out];
                out++;
                base++;
            } else {
                out++;
            }
        }

        return base;
    }
}

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.py
def getSeqLen(start, nums):
    seqLen = 1
    while start + 1 in nums:
        seqLen += 1
        start += 1

    return seqLen


class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        nums = set(nums)
        maxSeqLen = 0

        for n in nums:
            if n-1 not in nums:
                seqLen = getSeqLen(n, nums)

                maxSeqLen = max(maxSeqLen, seqLen)
            

        return maxSeqLen